### PR TITLE
Updated submit.rb - In development Slack should be off

### PR DESCRIPTION
**Pull request by tsrandrei**

### CODE REVIEW

File: app/operations/ops/idea/submit.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Submit < BaseOperation
       step Contract::Build(constant: ::Propose::ProposeForm)
       step Contract::Validate()
       step Contract::Persist()
-      success :send_message_to_slack!
+      success :send_message_to_slack! unless Rails.env.development?
```
Comment (Mehonoshin) [inline app/operations/ops/idea/submit.rb]: That's a bad idea :( The code itself should now know anything about the specific environment, that we are using.
Can you explain, why do you want to disable this for development?

Comment (Mehonoshin) [reply app/operations/ops/idea/submit.rb]: Let me reproduce it on my machine, it should work even without this patch.

Comment (tsrandrei) [reply app/operations/ops/idea/submit.rb]: https://monosnap.com/file/iq2siMzDj5LDk13VAOH8gFMjAvKgVS

Comment (Mehonoshin) [reply app/operations/ops/idea/submit.rb]: The reason is that you don't have redis running. It is required for running the application locally.

Comment (tsrandrei) [reply app/operations/ops/idea/submit.rb]: so, probably this should be into documentation also

**Pull request closed without merging**